lib/facebook.ex

```diff
@@ -83,6 +83,30 @@ defmodule Facebook do
   end
 
   @doc """
+  A Picture for a Facebook User
+
+  See: https://developers.facebook.com/docs/graph-api/reference/user/picture/
+  """
+  @spec picture(type :: String.t, access_token) :: response
+  def picture(type, access_token) do
+    picture(type, access_token, [])
+  end
+
+  @doc """
+  A Picture for a Facebook User
+
+  See: https://developers.facebook.com/docs/graph-api/reference/user/picture/
+  """
+  @spec picture(type :: String.t, access_token, options) :: response
+  def picture(type, access_token, options) do
+    fields = if !is_nil(Config.appsecret) do
+      [type: type, redirect: false, appsecret_proof: encrypt(access_token)]
```

Comment:
Isn't this line wrong because you overwrite the fields instead of appending the appsecret?